ablator/main/model/wrapper.py

```diff
@@ -486,6 +717,27 @@ def apply_loss(
         scaler: torch.cuda.amp.GradScaler,
         scheduler: ty.Optional[Scheduler],
     ) -> float | None:
+        """
+        Calculate the loss and apply the gradients, step the optimizer and scheduler.
```

Comment:
maybe change "step the optimizer and scheduler" to "call optimizer.step() and scheduler.step()"?